buildx.sh

```diff
@@ -69,22 +69,27 @@ main() {
             BUILD_ARGS=""
             ;;
     esac
-    IMAGE="privatebin/$IMAGE"
+    IMAGE="privatebin/${IMAGE}:"
```

Comment:
Is the colon a typo or deliberate was the first thought when I saw this, because a separator sign is of course not part of the `IMAGE` (name).

Seeing the line below it makes sense, I would, however, always write `:` explicitly, so it is more obvious what it is about.
Similar to how it is unwritten truth that you always end paths without a separator i.e. `/home/someuser` and then, if saved in a var, append that by the separator sign when used further, e.g. like this: `$homeDir/.bashrc`.

I.e.…
```suggestion
    IMAGE="privatebin/${IMAGE}"
```

and adjusting the code below.

Reply:
applied revert in b5915a063ab4d267beb94a3d66a0cdcc8b31cd1a - it looked cleaner to me, but that is why we do the reviews - it needs to make sense to other brains, too!